index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
+       <h4>The man who saved a billion lives</h4>
+       <div id="img-div">
+        <img id="image" src="images/img.jfif" alt="imagen de muestra">
+        <div id="img-caption">
+      <h4>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</h4>
+        </div>
+       </div>
```

Comment:
For a good lecture of your code use a right identation

Reply:
Please try to explain in more detail to the student why it is important to use indentation and try not to make any mistakes in spelling also. I think you wanted to write "indentation" instead of "identation" :+1: 
**ex:** Please use Proper Indentation of your code. It makes your code cleaner and more readable. It also makes it easier to prevent bugs.